.claude/CLAUDE.md

```
@@ -0,0 +1,260 @@
+
+## Claude Code 관련 질문 처리
+
+claude-code-guide는 틀린 답을 낼 때가 있다. 사용자가 Claude Code 기능에 대해 추가 질문을 하면, 공식 문서를 curl로 직접 참조해서 답한다.
+
+```bash
+curl https://code.claude.com/docs/ko/overview.md
+```
+
+문서 URL 패턴: `https://code.claude.com/docs/ko/{페이지명}.md`
+
+답변 후에는 `AskUserQuestion`으로 퀴즈를 내서 사용자가 직접 따라해보도록 안내한다.
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)